content/pt-br/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Entendendo os objetos do Kubernetes
+content_type: concept
+weight: 10
+card:
+ name: concepts
+ weight: 40
+---
+ 
+<!-- overview -->
+Esta página explica como os objetos do Kubernetes são representados na API do Kubernetes e como você pode expressá-los no formato `.yaml`.
+ 
+<!-- body -->
+## Entendendo os objetos do Kubernetes  {#kubernetes-objects}
+ 
+*Os objetos do Kubernetes* são entidades persistentes no sistema do Kubernetes. Usuários do Kubernetes essas entidades para representar o estado do cluster. Especificamente, eles podem descrever:
+ 
+* Quais aplicativos estão sendo executados (e em quais nós).
+* Os recursos disponíveis para esses aplicativos
+* As políticas acerca de como esses aplicativos se comportam, como políticas de reinicialização e tolerâncias a falhas.
+
+ 
+Um objeto do Kubernetes é um “registro de intenção”--uma vez criado o objeto, o sistema do Kubernetes trabalha constantemente para garantir que este objeto existe. Ao criar um objeto, você está efetivamente falando para o sistema do Kubernetes como você quer que a carga do seu cluster seja. Este é o *estado desejado* do seu cluster.
+ 
+Para trabalhar com objetos do Kubernetes seja para criar, modificar ou deletar eles, você precisará usar a [Kubernetes API](/docs/concepts/overview/kubernetes-api/). Quando você usa a interface de linha de comando do  `kubectl`, por exemplo, o CLI faz as chamadas necessárias da API do Kubernetes para você. Você também pode usar a API do Kubernetes diretamente no seu próprio programa usando um dos [Client Libraries](/docs/reference/using-api/client-libraries/).
+ 
+
+
+### Especificação e status do objeto
+ 
+Quase todos os objetos do Kubernetes incluem dois campos de objetos aninhados que governam a configuração do objeto: a *`especificação`* do objeto e o *`status`* do objeto. Para objetos que têm especificação, você tem que definir isso quando você cria o objeto, fornecendo uma descrição das características que você quer que o recurso tenha: o seu _estado desejado_.
```

Comment:
I guess you cannot use "bold" (*) and code marking (`) on the same sentence, I would use the bold marking here in "especificação" and "status"

Reply:
Hi, in the English documentation it is bold (*) with (`) do I just keep it bold?

"... the object *`spec`* and the object *`status`*. ..."